note

```diff
@@ -1,29 +1,38 @@
 #!/bin/bash
 
-create_note () {
-    if [ ! -f $note_path ]; then
-        mkdir -p ~/notes/$year/$month/
-        touch $note_path
-        printf "$day-$month-$year\n---\n\n" > $note_path
-        printf "Created new note: $note_path\n"
+YEAR=$(date +'%Y')
+MONTH=$(date +'%m')
+DAY=$(date +'%d')
+NOTE_DIR="$HOME/notes/$YEAR/$MONTH"
+NOTE_NAME="$DAY.md"
+
+NOTERC="$XDG_CONFIG_HOME/note/noterc"
```

Comment:
Should also probably add info about the noterc to the docs. I can work on that.

Reply:
There is a hint add the end of the readme